CodeStarterCamp_Week2/lotto.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  lotto.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 양주원 on 2023/03/15.
+//
+
+import Foundation
+
+
+func winLottoNumbers() -> Set<Int> {
+    var winningNumberSet: Set<Int> = Set<Int>()
```

Comment:
훨씬 좋은 네이밍이군요!
한가지만 추가적으로 말씀드리면, 특별한 몇몇 경우를 제외하고 스위프트에서는 타입명 자체를 변수명에 넣지 않습니다.
```suggestion
    var winningNumbers: Set<Int> = Set<Int>()
```
요정도로 표현하는게 좋겠네요